packages/office-ui-fabric-react/src/components/Dialog/Dialog.types.ts

```diff
@@ -21,6 +21,7 @@ export interface IDialogProps extends React.ClassAttributes<DialogBase>, IWithRe
   /**
    * Optional callback to access the IDialog interface. Use this instead of ref for accessing
    * the public methods and properties of the component.
+   * @deprecated Deprecating because the ref was returning an empty object given that IDialog interface is empty.
```

Comment:
Can this include a comment about what to use instead? Also should be present tense instead of past.

Reply:
I can change it to present tense but there is really nothing to use instead since the prop was not providing any functionality in the first place.

Reply:
Spoke with @jdhuntington and have updated the comment with his suggestion.